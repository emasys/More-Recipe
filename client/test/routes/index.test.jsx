import React from 'react';
import { shallow } from 'enzyme';

// Component
import Routes from '../../src/routes/AppRoutes';

let wrapper = null;

describe('Test suite for routes', () => {
  beforeEach(() => {
    wrapper = shallow(<Routes />);
  });
  it('Should render without errors', () => {
    expect(wrapper.length).toEqual(1);
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
