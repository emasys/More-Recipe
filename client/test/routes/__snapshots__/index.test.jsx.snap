// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for routes should match snapshot 1`] = `
<BrowserRouter>
  <div>
    <Connect(Preloader) />
    <Switch>
      <Route
        component={[Function]}
        path="/signin"
      />
      <Route
        component={[Function]}
        path="/signup"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/catalog"
      />
      <Route
        component={[Function]}
        path="/recipe/:id"
      />
      <Route
        component={[Function]}
        path="/user/:id"
      />
      <Route
        component={[Function]}
        path="/manageUsers"
      />
      <Route
        component={[Function]}
        path="/favorites"
      />
      <Route
        component={[Function]}
        path="/new"
      />
      <Route
        component={[Function]}
        path="/profile/:id"
      />
      <Route
        component={[Function]}
        path="/category/:cat"
      />
      <Route
        component={[Function]}
        path="*"
      />
    </Switch>
    <Footer />
  </div>
</BrowserRouter>
`;
